### PR TITLE
Build amd64 and arm64/v8 Docker Images

### PR DESCRIPTION
**Pull request by drcapulet**

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -32,9 +32,24 @@ changelog:
   use: github
 
 dockers:
+  - build_flag_templates:
+      - "--platform=linux/amd64"
+    image_templates:
+      - "ghcr.io/persona-id/{{.ProjectName}}:{{ .Tag }}-amd64"
+    goarch: arm64
```
Comment (derektamsen) [inline .goreleaser.yaml:39]: I think `goarch` should be amd64 or a reference to the x86 arch?

File: .goreleaser.yaml [hunk 2]
```diff
@@ -32,9 +32,24 @@ changelog:
   use: github
 
 dockers:
+  - build_flag_templates:
+      - "--platform=linux/amd64"
```
Comment (derektamsen) [inline .goreleaser.yaml:36]: The [docs](https://goreleaser.com/cookbooks/multi-platform-docker-images/) also include `--pull` in the build flags. Maybe to ensure you use layer caches if they are available?

File: .goreleaser.yaml [hunk 3]
```diff
@@ -32,9 +32,24 @@ changelog:
   use: github
 
 dockers:
+  - build_flag_templates:
```
Comment (derektamsen) [inline .goreleaser.yaml:35]: The [docs](https://goreleaser.com/cookbooks/multi-platform-docker-images/) also mention it's important to include `use: buildx`

File: .goreleaser.yaml [hunk 4]
```diff
@@ -32,9 +32,24 @@ changelog:
...
+      - "--platform=linux/amd64"
+    image_templates:
+      - "ghcr.io/persona-id/{{.ProjectName}}:{{ .Tag }}-amd64"
+    goarch: arm64
+    skip_push: false
```
Comment (derektamsen) [inline .goreleaser.yaml:40]: I wonder if you set `skip_push: true` and as long as the `image_template` combines them you can push a single multi-arch tag? Might be cleaner?

File: .goreleaser.yaml (hunk 1 above)
Comment (drcapulet) [reply .goreleaser.yaml:39]: Yeap, I must have copied back because this was the original block.

File: .goreleaser.yaml (hunk 2 above)
Comment (drcapulet) [reply .goreleaser.yaml:36]: Will add - I must have missed that in the original Docker build.

File: .goreleaser.yaml (hunk 3 above)
Comment (drcapulet) [reply .goreleaser.yaml:35]: That doc is much more useful than the [one](https://goreleaser.com/customization/docker_manifest/) I was reading / copying from.

File: .goreleaser.yaml (hunk 4 above)
Comment (drcapulet) [reply .goreleaser.yaml:40]: The `skip_push` option seems to be related to not pushing tags that look like release candidates - I think the tag with the arch is just a requirement of how it references them but it'd be nice to not have them and just have the single tag.

Comment (derektamsen) [reply .goreleaser.yaml:42]: Bummer, I see notices on the `docker_manifests` docs that state the image needs to be pushed.

**Review by derektamsen — APPROVED** [review]

**Pull request merged**